doc/input-output-reference/src/overview/group-design-objects.tex

```diff
@@ -891,7 +891,7 @@ \subsubsection{Inputs}\label{inputs-012}
 
 This field is the name of schedule that defines how outdoor air requirements change over time. The field is optional. If left blank, the schedule defaults to 1.0. If used, then the schedule values are multiplied by the outdoor air flow rate defined by the previous fields. The schedule values must be between 0 and 1, inclusive.  
 
-If this DesignSpecification:OutdoorAir object is referenced by a Controller:MechanicalVentilation object (either directly or indirectly through Sizing:Zone), the schedule will be applied to all types of outdoor air calculations for the corresponding zone, regardless of the System Outdoor Air Method selected. If the schedule value is zero, then the zone will be completely removed from the system outdoor air calcaulations.
+If this DesignSpecification:OutdoorAir object is referenced by a Controller:MechanicalVentilation object (either directly or indirectly through Sizing:Zone), the schedule will be applied to all types of outdoor air calculations for the corresponding zone, regardless of the System Outdoor Air Method selected. If the schedule value is zero, then the zone will be completely removed from the system outdoor air calculations.
 
```

Comment:
@mjwitte Let me remove the paragraph I just added. Here is an issue I think. The Controller;MechanicalVentilation for a system and DesignSpecification:OutdoorAir for a terminal should be interlocked. We should not let these objects fight each other internally and give users a clear picture how to make correct inputs of OA.